aten/src/ATen/core/Dict.h

```diff
@@ -338,6 +338,25 @@ class Dict final {
    */
   void reserve(size_type count) const;
 
+  /**
+   * Value equality comparison. This function implements Python-like semantics for
+   * equality: two dicts with the same identity (e.g. same pointer) trivially
+   * compare equal, otherwise each element is compared for equality.
+   */
+  template <class Key_, class Value_>
+  friend bool operator==(
+      const Dict<Key_, Value_>& lhs,
+      const Dict<Key_, Value_>& rhs);
+  template <class Key_, class Value_>
+  friend bool operator!=(
```

Comment:
notice `!=` doesn't have an impl or test, maybe that's intended?

Reply:
good catch! will add